Dockerfile

```diff
@@ -4,6 +4,9 @@ WORKDIR /app/
 COPY src /app/src
 COPY requirements.txt /app/requirements.txt
 
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends gcc
```

Comment:
why do we need gcc?

Reply:
It was otherwise throwing an error while building the Docker image. Strange, but i had to install gcc to fix it